### DISCUSSION TITLE
How to fix the error caused by SSL: EOF occurred in violation of protocol    如何修复SSL带来的错误：EOF occurred in violation of protocol

### DISCUSSION THREAD

**Discussion by xinlingqudongX** [encode/httpx#1616] [Potential Issue]
The starting point for issues should usually be a discussion...

https://github.com/encode/httpx/discussions

Possible bugs may be raised as a "Potential Issue" discussion, feature requests may be raised as an "Ideas" discussion. We can then determine if the discussion needs to be escalated into an "Issue" or not.

This will help us ensure that the "Issues" list properly reflects ongoing or needed work on the project.

---
httpx:0.18.0

- 我使用httpx访问https协议的网站触发了SSL的错误，我的系统是windows 10，并且使用了系统代理
- I use httpx to access the https protocol website triggering an SSL error, my system is windows 10, and the system proxy is used

- 我使用requests访问网络也遇到了相同的错误，但是我将urllib3更新到指定的版本之后就跳过了错误，但是使用httpx并没有任何改变
- I used requests to access the network and encountered the same error, but after I updated urllib3 to the specified version, I skipped the error, but using httpx did not change anything.

- 我该如何修复这个错误呢？谢谢。
- How can I fix this error? Thank you.
![Snipaste_2021-04-28_23-54-33](https://user-images.githubusercontent.com/26372348/116436499-0a2d8380-a87f-11eb-938b-eb60b01cb97e.png)
![Snipaste_2021-04-28_23-52-08](https://user-images.githubusercontent.com/26372348/116436506-0bf74700-a87f-11eb-8de2-b3d868cd9eb7.png)

**Comment by lovelydinosaur** [comment]
Hiya! 👋

Are you able to share the URL that you're making a request against?
We'll need to be able to replicate the issue ourselves in order to be able to help you.

**Reply by xinlingqudongX** [reply]
我想我已经找到问题了，因为我的VPN代理程序配置错误的原因，导致出现了这个错误，现在已经可以请求了，😀谢谢你的帮助！
I think I have found the problem. This error occurred because of the wrong configuration of my VPN proxy program. It is now possible to request, 😀 Thank you for your help！
